src/rules/importBlacklistRule.ts

```diff
@@ -54,9 +88,140 @@ export class Rule extends Lint.Rules.AbstractRule {
 }
 
 function walk(ctx: Lint.WalkContext<string[]>) {
+    type Options = Array<string | { [moduleName: string]: string[] }>;
```

Comment:
You can move `Options` out from `walk` function and correctly type `options` key without casting on line 99.

```
function walk(ctx: Lint.WalkCONTEXT< Array<string | { [moduleName: string]: string[] }>>) {
```